addons/dogtags/XEH_postInit.sqf

```diff
@@ -7,6 +7,11 @@
 if (hasInterface || isServer) then {
     [QGVAR(broadcastDogtagInfo), {
         GVAR(dogtagsData) set _this;
+        
+        if (isNil "CBA_fnc_renameInventoryItem") exitWith {}; // requires https://github.com/CBATeam/CBA_A3/pull/1329
+        params ["_item", "_dogTagData"];
+        private _name = [LLSTRING(itemName), ": ", _dogtagData param [0, LELSTRING(common,unknown)]] joinString "";
```

Comment:
The code below is in line with the current code on master (I changed it in #10096)
```suggestion
        private _name = _dogtagData param [0, ""];

        // If data doesn't exist or body has no name, set name as "unknown"
        if (_name == "") then {
            _name = LELSTRING(common,unknown);
        };
        
        _name = [LLSTRING(itemName), ": ", _name] joinString "";
```